test/e2e/operator.go

```
@@ -329,6 +212,150 @@ func TestKCMRecovery(t *testing.T) {
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if createPod {
+		err = podCreate(kubeClient, name, labels)
+		if err != nil {
+			t.Fatal(err)
+		}
+		var pods *corev1.PodList
+		wait.PollImmediate(time.Second*1, testTimeout, func() (bool, error) {
```

golangci-lint (2.11.4)

[error] Error return value of `wait.PollImmediate` is not checked

(errcheck)